apps/passport-server/src/services/issuanceService.ts

```diff
@@ -159,7 +145,7 @@ export class IssuanceService {
               }
               const { pcd } = await verifyFeedCredential(
                 req.pcd,
-                this.cachedVerifySignaturePCD.bind(this)
+                this.cachedVerifySignaturePCD
```

Comment:
are you sure you wanted to get rid of the `.bind` here and elsewhere?

Reply:
This is now bound in the constructor which is a better practice so we don't accidentally forget to bind this at callsite. 
https://github.com/proofcarryingdata/zupass/pull/1131/files#diff-fa355606c55306593a2a38a10ebb5c0eae516e448da069a0ed223baefd50a3edR132

Also, it looks like the more up to date practice is use arrow function but I don't know if we like mix function styles or ready to refactor.